lighthouse-core/formatters/partials/critical-request-chains.html

```diff
@@ -89,7 +95,8 @@
         <span class="cnc-node__tree-hostname">({{ this.hostname }})</span>
       {{/parseURL}}
       {{#unless hasChildren}}
-        - <span class="cnc-node__chain-duration">{{chainDuration startTime this.node.request.endTime }}ms, {{formatTransferSize this.transferSize}}KB</span>
+        - <span class="cnc-node__chain-duration">{{chainDuration startTime this.node.request.endTime }}ms</span>,
+          <span class="cnc-node__chain-duration">{{formatTransferSize this.transferSize}}KB</span>
```

Comment:
split into two so that the comma wasn't bolded. and yah im lazily not adjusting the classname. i could.